include/RAJA/policy/MultiPolicy.hpp

```diff
@@ -170,13 +170,8 @@ struct policy_invoker<0, size, Policy, rest...> {
   void invoke(int offset, Iterable &&iter, Body &&body)
   {
     if (offset == size - 1) {
-      util::PluginContext context{util::make_context<Policy>()};
-      util::callPreLaunchPlugins(context); 
-
       using policy::multi::forall_impl;
       forall_impl(_p, iter, body);
-
-      util::callPostLaunchPlugins(context);
```

Comment:
These caused the plugins to be called twice, does this seem reasonable?

Reply:
I think this is reasonable, it makes sense that plugins should be called at the more generic top level forall call.

Reply:
Yes, as long as the result is we always get a single set for all code paths that's what we want.